### ISSUE TITLE
Cant build from master

### ISSUE THREAD

**Issue by SorMV** [sanity-tool/bitreader-service#12]
https://pp.userapi.com/c851336/v851336869/88387/YrO8Viiuwew.jpg

**Comment by okutane** [comment]
Indeed, the build is broken: https://travis-ci.org/sanity-tool/bitreader-service/builds/471854380

Thanks for reporting this, @SorMV !
